apps/src/studio/rocketHeightLogic.js

```diff
@@ -0,0 +1,51 @@
+var CustomGameLogic = require('./customGameLogic');
+var studioConstants = require('./constants');
+var Direction = studioConstants.Direction;
+var codegen = require('../codegen');
+var api = require('./api');
+
+/**
+ * Custom logic for the Rocket Height levels
+ * @constructor
+ * @implements CustomGameLogic
+ */
+var RocketHeightLogic = function (studio) {
+  CustomGameLogic.apply(this, arguments);
+  this.rocketIndex = 0;
+  this.rocket = null;
+  this.last = Date.now();
+  this.seconds = 0;
+  this.height = 0;
+};
+RocketHeightLogic.inherits(CustomGameLogic);
+
+RocketHeightLogic.prototype.onTick = function () {
+  
+  // Update the rocket once a second
+  if (Date.now() - this.last < 1000) {
+    return;
+  } else {
+    this.last = Date.now();
+    this.seconds++;
```

Comment:
I suspect we don't really care, but the way this is set up, seconds is not guaranteed to be accurate. For example, if I call onTick once every 1.5 seconds, after 1 tick 1.5 seconds will have elapsed and seconds will be 1. After 2 ticks, 3 seconds will have elapsed and seconds will be 2.
In practice, I think we call onTick many times per second, such that this is even less likely to be a problem unless precision is really important.